TypeCobol.Test/Parser/Programs/Cobol2002/LevelChecker.rdz.tcbl

```diff
@@ -40,10 +40,4 @@
 
        PROCEDURE DIVISION.
 
-      *KO Level 88 under type bool in parameter
-       DECLARE PROCEDURE MyProc
-         INPUT exists TYPE BOOL
-                88 exists-Ok value 'O'.
-       END-DECLARE.
```

Comment:
I think we should keep the test and simply update the error message. It is still an invalid syntax, only the way to check it has changed.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/3c0f9d082a18ff1b641cb676ce9152a389f4f984